.env.testing

```
@@ -0,0 +1,3 @@
+DB_CONNECTION=sqlite
+DB_DATABASE=:memory:
+APP_KEY=base64:SomeGeneratedKey==
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_KEY key should go before the DB_CONNECTION key

(UnorderedKey)